packages/apptive_grid_form/lib/src/widgets/apptive_grid_form_widgets.dart

```diff
@@ -21,3 +21,4 @@ export 'package:apptive_grid_form/src/widgets/form_widget/signature_form_widget.
 export 'package:apptive_grid_form/src/widgets/form_widget/text_form_widget.dart';
 export 'package:apptive_grid_form/src/widgets/form_widget/uri_form_widget.dart';
 export 'package:apptive_grid_form/src/widgets/form_widget/user_form_widget.dart';
+export 'package:apptive_grid_form/src/widgets/form_widget/form_content_widgets.dart';
```

Comment:
The whole point of moving the file to a different file was to not export the widgets to outside of the package.

Reply:
Is there a reason not to expose/export it? Is seem to me like there is not really a downside to it and it might help for testing inside of apps using the package.

Reply:
Exposing them means that doing (breaking) Api Changes is far more likely to happen. 
Accessing them outside of the app will still always be possible (as they are public classes) by strictly typing out the `/src/` base path however it will rightfully produce a lint warning as these widgets are not part of the public facing API of this package

Reply:
Ok, that makes sense. I removed the export.